packages/service-library/src/web-api/scan-notification-error-codes.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// tslint:disable: no-unnecessary-class variable-name
+
+export declare type ScanNotificationErrorCodeName = 'InternalError' | 'NotificationError';
+
+export interface ScanNotificationErrorCode {
+    // This type is part of the REST API client response.
+    // Ensure compatibility when changing this type.
+    code: ScanNotificationErrorCodeName;
+    codeId: number;
+    message: string;
+}
+
+export class ScanNotificationErrorCodes {
+    public static NotificationError: ScanNotificationErrorCode = {
+        code: 'NotificationError',
```

Comment:
you should also be sending the last notification url response code. (similar to how we do it for scan)